strawberry/schema/schema_converter.py

```diff
@@ -468,9 +468,22 @@ def _get_arguments(
             info: Info,
             kwargs: Dict[str, Any],
         ) -> Tuple[List[Any], Dict[str, Any]]:
+            # FIXME: An extension might have asked the field to ignore the resolver
+            # arguments, but we need them here since we are calling it.
+            # This is a bit of a hack, but it's the easiest way to get the arguments
+            # This happens in mutation.InputMutationExtension
+            field_arguments = field.arguments
+            if field.base_resolver:
+                existing = [arg.python_name for arg in field_arguments]
+                field_arguments = field_arguments + [
+                    arg
+                    for arg in field.base_resolver.arguments
+                    if arg.python_name not in existing
+                ]
+
```

Comment:
See my above comment on `field.arguments`. Maybe we can find a solution that fixes this in the same sweep 😊

Reply:
I still had to keep this code even after changing `StrawberryField.arguments`. Don't know if there's a more elegant solution for this atm though